Tests/LinkCommandsTest/main.py

```diff
@@ -0,0 +1,219 @@
+import argparse
```

Comment:
Could you please move it out to this repository https://github.com/Software-Analysis-Team/link-commands